client/state/immediate-login/constants.js

```diff
@@ -0,0 +1,5 @@
+/** @format */
+
+export const REASON_MANUAL_RENEWAL = 'manual_renewal';
+
+export const SAVE_IMMEDIATE_LOGIN_INFORMATION = 'SAVE_IMMEDIATE_LOGIN_INFORMATION';
```

Comment:
This should probably be defined in `client/state/action-types.js` instead, and also use a name that puts the noun in front (maybe `IMMEDIATE_LOGIN_STATUS_SAVE`?) - see https://wpcalypso.wordpress.com/devdocs/docs/our-approach-to-data.md#actions